packages/react-composites/src/composites/CallComposite/styles/ScreenSharePopup.styles.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTheme, IButtonStyles, IModalStyleProps, IModalStyles, IStyleFunctionOrObject } from '@fluentui/react';
+import { CSSProperties } from 'react';
+
+const palette = getTheme().palette;
+
+export const screenSharePopupModalStyles: IStyleFunctionOrObject<IModalStyleProps, IModalStyles> = {
+  root: { width: '100%', height: '100%' },
+  main: {
+    boxShadow: '0px 1.2px 3.6px rgba(0, 0, 0, 0.1), 0px 6.4px 14.4px rgba(0, 0, 0, 0.13);',
```

Comment:
You can use theme here as well.
```tsx
import { getTheme } from '@fluentui/react';
const theme = getTheme();
{
  boxShadow: theme.effects.elevation8
}
```

Reply:
Thanks Anjul, changed the two boxShadows to elevations from the theme. They are not exactly to the same as the design but I cannot tell any difference visually.